src/templates/daily_report/html.handlebars

```diff
@@ -0,0 +1,517 @@
+        */
+        #utility div{
+            /*@editable*/ text-align:center;
+        }
+
+        #monkeyRewards img{
+            max-width:190px;
+        }
+    </style>
+</head>
+<body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0" style="background-color: #eeeeee">
+<center>
+    <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="backgroundTable">
+        <tr>
+            <td align="center" valign="top" style="padding-top:20px;">
+                <table border="0" cellpadding="0" cellspacing="0" width="600" id="templateContainer">
+                    <tr>
+                        <td align="center" valign="top">
+                            <!-- // Begin Template Header \\ -->
+                            <table border="0" cellpadding="0" cellspacing="0" width="600" id="templateHeader">
+                                <tr>
+                                    <td class="headerContent" align="center">
+
+                                        <!-- // Begin Module: Standard Header Image \\ -->
+                                        <img src="http://www.openhim.org/wp-content/uploads/2015/01/opehim-logo-green.png" style="max-width:300px;" id="headerImage campaign-icon" mc:label="header_image" mc:edit="header_image" mc:allowdesigner mc:allowtext />
+                                        <!-- // End Module: Standard Header Image \\ -->
+
+                                    </td>
+                                </tr>
+                            </table>
+                            <!-- // End Template Header \\ -->
+                        </td>
+                    </tr>
+                    <tr>
+                        <td align="center" valign="top">
+                            <!-- // Begin Template Body \\ -->
+                            <table border="0" cellpadding="0" cellspacing="0" width="800" id="templateBody">
+                                <tr>
+                                    <td valign="top">
+                                        <!-- // Begin Module: Standard Content \\ -->
+                                        <table border="0" cellpadding="20" cellspacing="0" width="100%">
+                                            <tr>
+                                                <td valign="top" align="center" class="bodyContent">
+                                                    <div mc:edit="std_content00">
+                                                        <h1 class="h1" style="color: darkgreen">{{type}} Transaction  Report for {{instance}}</h1>
+                                                    </div>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td valign="top" style="padding-top:0; padding-bottom:0;">
```

Comment:
I think it would be better to put these styles in to css classes rather as they repeat a lot. You are already using a css script above.